lm_eval/tasks/schemingtasks/scheming_dataset_questions.csv

```diff
@@ -0,0 +1,13 @@
+added_by,subject,scenario,free_response_prompt,mc_prompt,1,2,answer,explanation,goal_question,goal_answer,context_question,context_answer

```

Comment:
If this PR is intended to be merged, data should be hosted on HuggingFace datasets rather than a CSV in this repo!

Reply:
CSVs removed and dataset added to HF